classes/fields/datetime.php

```diff
@@ -446,12 +437,25 @@ public function format_value_display( $value, $options, $js = false ) {
 	 */
 	public function format_display( $options, $js = false ) {
 
-		if ( 'custom' !== pods_v( static::$type . '_type', $options, 'format' ) ) {
+		if ( 'custom' === pods_v( static::$type . '_type', $options, 'format' ) ) {
+			if ( $js ) {
+
+				// Gets format strings in jQuery UI format.
+				$date = $this->format_date( $options, $js );
+				$time = $this->format_time( $options, $js );
+
+				// Convert them to PHP date format.
+				$date = $this->convert_format( $date, array( 'source' => 'jquery_ui', 'type' => 'date' ) );
+				$time = $this->convert_format( $time, array( 'source' => 'jquery_ui', 'type' => 'time' ) );
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: acb56a7caac6be55692c7311da88a47b</sub>